docs/getting-started/cronos-smart-contract.md

```diff
@@ -155,7 +155,7 @@ By default, the script will be using your local host `"127.0.0.1"`  - If you are
       accounts: getHDWallet(),
      },
     testnet: {
-      url: "https://evm-t3.cronos.org/:8545",
+      url: "https://cronos-testnet-3.crypto.org:8545",
```

Comment:
```suggestion
      url: "https://evm-t3.cronos.org:443",
```
Thanks for bringing it up. 
`https://cronos-testnet-3.crypto.org:8545/` is the old URL and it may be removed in the near future. It is now `https://evm-t3.cronos.org/` and the port is the TLS default port `443`.